docs/spring.md

```diff
@@ -25,8 +27,19 @@ Some additional dependencies:
 For testing we add:
 * `spring-boot-starter-test` - that adds capabilities to create Spring unit and integration tests,
   based on [jUnit 5 (Jupiter)](https://junit.org/junit5/docs/current/user-guide/) platform.
-* `testcontainers` - will automatically start a PostgreSQL in docker container during testing
+* `testcontainers` - version management bom
+
+The second plugin named `spring-db` adds dependencies, to enable your application to use
```

Comment:
It is good to add this plugin in the README at the root level too - 
there is only spring plugin there.

Reply:
good catch!
done